### ISSUE TITLE
The Site

### ISSUE THREAD

**Issue by AvgZing** [BoxOfDevs/SwitchFromLeet#5]
The index page will have four options: 
1. Explain the advantages ans disadvantages of leet, hosting yourself, using a VPS or using MCPE hosting (explain why leet is bad to devs)
1. Calculate how much money you could safe by switching to a selected option (Yourself, VPS, MCPE); based on JS and JQuery
1. Host it yourself
1. Use an online hoster

## For Host It Yourself

1. Display the pros, cons, and features of PMMP vs Tesseract (The two current leading Softwares)
2. Describe how to set up each of them
3. Describe how to add plugins
4. Describe settings
5. Describe how much more you can customize the settings
6. Describe how much cheaper it is to download plugins for free or simply commission a totally-unique one from someone
7. Tell them how to get a SUPER CUSTOM IP with no-ip or freenom for free, or namecheap for payed-- All of which you cannot do with leet

## For Use Online Hoster

1. Same as the "For Host It Yourself" numbers 1-6 (Dont forget to include in cons how most online hosters do NOT Give a custom port.
2. Show how it is almost as simple as Leet for setting up and adding plugins and changing settings and restarting servers
3. Tell them how to get a SUPER CUSTOM IP with no-ip or freenom for free, or namecheap for payed-- All of which you cannot do with leet

## ToDo

* [ ] Team up with hosting providers to provide coupons for from-leet-switchers.

**Comment by AvgZing** [comment]
@HimbeersaftLP I dont have time to actally make this website and IDK if u r gonna use a template or make it urself, I just included the basics and few specifics here

**Comment by lavender-sys** [comment]
@HimbeersaftLP likes HTML5 UP, so he's probley using that.

**Comment by AvgZing** [comment]
Lol kk

**Comment by AvgZing** [comment]
Personally, I like html5up.net and free-css.com/free-css-templates

**Comment by AvgZing** [comment]
I think ~~Mr.German Rasberry Juice~~ Himbeer is busy so I will just go ahead...

**Comment by HimbeersaftLP** [comment]
> I think Mr.German Rasberry Juice Himbeer is busy so I will just go ahead...

Sure, I updated the ToDo list a bit. Also, if you are using a template, please don't forget to give the creator credit = Leave the footer where it is!

**Comment by Vaults1** [comment]
or just make it ourself lol

**Comment by AvgZing** [comment]
@HimbeersaftLP no. pixelarity.com says its attribution free-- and I bought it. So HAH!

**Comment by HimbeersaftLP** [comment]
> @HimbeersaftLP no. pixelarity.com says its attribution free-- and I bought it. So HAH!

Which website are you talking about?

**Comment by AvgZing** [comment]
for this one. The template that I used... I used Stellar by HTML5 Up, bought fromm pixelarity.com..

**Comment by lavender-sys** [comment]
Why buy it when it's FREE at html5up.net?

**Comment by AvgZing** [comment]
Because it's attribution free on pixelarity, on html5up.net you need to keep the footer.

PS: attribution free means that you don't have to give credit to the author, you can take the credit for yourself.

**Comment by Taki21** [comment]
hi guys! its me! ill be more active during april break so ask me 4 help if u need any lol
also next year ill be more active (from sepember 2017)

**Comment by AvgZing** [comment]
Great! Please register on BOD Discord ASAP taking plz
